Makefile

```diff
@@ -0,0 +1,36 @@
+# Makefile for managing Jekyll project for blackpythondevs.github.io
+
+# Set up the environment by pulling the latest Ruby build definitions, installing Ruby 3.3.5,
+# as seen in the .ruby-version file, installing required gems and Python packages,
+# and configuring pre-commit hooks
+
+setup:
+    # Pull the latest ruby-build plugin updates
+	- git -C /root/.rbenv/plugins/ruby-build pull
```

Comment:
why did you choose rbenv versus one of the other ruby options

Reply:
This is done.

Reply:
I chose to choose rbenv because I noticed that it was already being used in the codebase and since I am familiar with it, it made sense to continue its usage. rbenv helps to maintain a ruby version in my repository and I wouldn't need to alter the system wide ruby version.